docs/tenants.md

```diff
@@ -0,0 +1,2 @@
+# **Tenants**
+Tenants allow projects spanning multiple bitbucket projects, teams and actual business groupings to be connected. 
```

Comment:
A whole page for this? :P

Reply:
Maybe use a little blurb below the command/user guide descriptions. The same way we add side information for see this command reference here thing with this blurb?

Reply:
Don't want the tenants to be upset :P